gcp/appengine/frontend3/src/templates/home.html

```diff
@@ -199,10 +201,54 @@ <h3 class="code-card-title">Scan directory recursively</h3>
       </div>
       <div class="mdc-layout-grid__cell--span-12">
         <div class="cta">
-          <a class="cta-primary link-button" href="https://github.com/google/osv-scanner">More details</a>
+          <a class="cta-primary link-button" href="https://google.github.io/osv-scanner/">More details</a>
         </div>
       </div>
     </div>
+    <div id="use-remediation-tools" class="mdc-layout-grid__cell--span-12 usage-examples">
+      <!-- &#8209; is a non breaking hyphen -->
+      <h2 class="heading">Remediation Tools</h2>
+      <div class="code-card-container mdc-layout-grid__inner">
+        <div class="code-card mdc-layout-grid__cell--span-6 mdc-layout-grid__cell--span-12-tablet">
+          <h3 class="code-card-title">Guided Remediation (basic)</h3>
+          <pre class="code-card-content" id="example-guided-remediation-basic">
+osv-scanner fix --non-interactive --strategy=in-place -L path/to/package-lock.json
+osv-scanner fix --non-interactive --strategy=relock -M path/to/package.json -L path/to/package-lock.json
+          </pre>
+          <clipboard-copy class="code-card-copy" for="example-guided-remediation-basic">
+            <mwc-icon-button class="icon" icon="content_copy"></mwc-icon-button>
+          </clipboard-copy>
+        </div>
+        <div class="code-card mdc-layout-grid__cell--span-6 mdc-layout-grid__cell--span-12-tablet">
+          <h3 class="code-card-title">Guided Remediation (interactive)</h3>
+          <pre class="code-card-content" id="example-guided-remediation-advanced">
+osv-scanner fix -M path/to/package.json -L path/to/package-lock.json
+          </pre>
+          <clipboard-copy class="code-card-copy" for="example-guided-remediation-advanced">
+            <mwc-icon-button class="icon" icon="content_copy"></mwc-icon-button>
+          </clipboard-copy>
+        </div>
+      </div>
+      <div class="mdc-layout-grid__cell--span-12">
+        <div class="cta">
+          <a class="cta-primary link-button" href="https://google.github.io/osv-scanner/experimental/guided-remediation/">More details</a>
+        </div>
+      </div>
+    </div>
+    <div id="use-the-github-action" class="mdc-layout-grid__cell--span-12 github-action">
+      <h2 class="heading">GitHub Action (OSV&#8209;Scanner)</h2>
```

Comment:
nit: we can just call this "GitHub Workflows"

Reply:
Updated via 4d3076cc31c101dde0b31dd7ca54828b04514d94